pkg/api/object_test.go

```diff
@@ -571,18 +571,22 @@ func TestListBuckets(t *testing.T) {
 	assert.Equal(t, `[{"name":"foo","owner":"bob","createdAt":"2016-08-05T16:23:34.343343Z","size":9,"numberOfObjects":6}]`, w.Body.String())
 
 	// Two buckets
-	timeThrough := 0
+	first = true
 	w = runTest(func(args ...string) (string, error) {
-		timeThrough++
-		if timeThrough == 1 {
+		if first {
+			first = false
```

Comment:
for maintainability, it may be useful to add a comment for each branch here about why you expect the 1st call to do x and why you expect the other calls to do y